### PR TITLE
Optimize Settings for BlueR and Sapphire Pro

### PR DESCRIPTION
**Pull request by ils15**
Changelog

Deactivate Z_AFTER_HOMING after some users request
Increase Hotend Idle Timeout (15min) to avoid preheat bugs
Put Touchscreen parameters inside configuration folder to prevent people to make screen calibration in first use

### CODE REVIEW

File: config/examples/Two Trees/BlueR/BlueR V1/Configuration.h [hunk 1]
```diff
@@ -2475,10 +2475,33 @@
...
+  #ifndef XPT2046_Y_CALIBRATION
+    #define XPT2046_Y_CALIBRATION         -12234
+  #endif
+  #ifndef XPT2046_X_OFFSET
+    #define XPT2046_X_OFFSET                 -45
+  #endif
+  #ifndef XPT2046_Y_OFFSET
+   #define XPT2046_Y_OFFSET                  349
+  #endif
+#elif ENABLED(TFT_CLASSIC_UI)
```
Comment (rhapsodyv) [inline config/examples/Two Trees/BlueR/BlueR V1/Configuration.h]: You don't need different xpt values to `TFT_CLASSIC_UI` anymore

File: config/examples/Two Trees/BlueR/BlueR V2/Configuration.h [hunk 2]
```diff
@@ -2480,10 +2480,33 @@
...
+  #ifndef XPT2046_Y_CALIBRATION
+    #define XPT2046_Y_CALIBRATION         -12234
+  #endif
+  #ifndef XPT2046_X_OFFSET
+    #define XPT2046_X_OFFSET                 -45
+  #endif
+  #ifndef XPT2046_Y_OFFSET
+   #define XPT2046_Y_OFFSET                  349
+  #endif
+#elif ENABLED(TFT_CLASSIC_UI)
```
Comment (rhapsodyv) [inline config/examples/Two Trees/BlueR/BlueR V2/Configuration.h]: same here

File: config/examples/Two Trees/BlueR/BlueR V3/Configuration.h [hunk 3]
```diff
@@ -2479,10 +2479,33 @@
...
+  #ifndef XPT2046_Y_CALIBRATION
+    #define XPT2046_Y_CALIBRATION         -12234
+  #endif
+  #ifndef XPT2046_X_OFFSET
+    #define XPT2046_X_OFFSET                 -45
+  #endif
+  #ifndef XPT2046_Y_OFFSET
+   #define XPT2046_Y_OFFSET                  349
+  #endif
+#elif ENABLED(TFT_CLASSIC_UI)
```
Comment (rhapsodyv) [inline config/examples/Two Trees/BlueR/BlueR V3/Configuration.h]: here too

File: config/examples/Two Trees/Sapphire Pro/Configuration.h [hunk 4]
```diff
@@ -2479,10 +2479,33 @@
...
+  #ifndef XPT2046_Y_CALIBRATION
+    #define XPT2046_Y_CALIBRATION         -12234
+  #endif
+  #ifndef XPT2046_X_OFFSET
+    #define XPT2046_X_OFFSET                 -45
+  #endif
+  #ifndef XPT2046_Y_OFFSET
+   #define XPT2046_Y_OFFSET                  349
+  #endif
+#elif ENABLED(TFT_CLASSIC_UI)
```
Comment (rhapsodyv) [inline config/examples/Two Trees/Sapphire Pro/Configuration.h]: here too

**Comment by ils15** [comment]
Ok, fixed it

**Pull request merged**